utility/src/main/java/org/oppia/android/util/parser/html/BUILD.bazel

```diff
@@ -0,0 +1,91 @@
+"""
+General-purpose utilities for parsing & rendering HTML within the app.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "html_parser",
+    srcs = [
+        "HtmlParser.kt",
+    ],
+    visibility = ["//app:__pkg__"],
+    deps = [
+        ":custom_bullet_span",
+        ":custom_html_content_handler",
+        ":tag_handlers",
+        "//third_party:androidx_core_core-ktx",
+        "//third_party:javax_inject_javax_inject",
+        "//utility/src/main/java/org/oppia/android/util/logging:console_logger",
+        "//utility/src/main/java/org/oppia/android/util/parser/image:url_image_parser",
+    ],
+)
+
+kt_android_library(
+    name = "tag_handlers",
+    srcs = [
+        "BulletTagHandler.kt",
+        "ConceptCardTagHandler.kt",
+        "ImageTagHandler.kt",
+        "MathTagHandler.kt",
+    ],
+    visibility = [
+        "//domain:__pkg__",
+        "//testing:__pkg__",
```

Comment:
Why these visibilities? If this is needed by tests, then we can use testing_visibility. However, a quick search didn't tell show me any classes in domain or testing that depends on this library. Am i missing something?

Reply:
used testing visibility